cmd/ferretdb/pg.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"time"
+
+	"github.com/FerretDB/FerretDB/internal/handlers"
+	"github.com/FerretDB/FerretDB/internal/handlers/pg"
+	"github.com/FerretDB/FerretDB/internal/handlers/pg/pgdb"
+)
+
+func init() {
```

Comment:
Same here. I believe, this coding choice needs to be documented somewhere. Otherwise, one day someone might want to get rid of those `init`s :)

I guess, the idea is that we can exclude various handlers from the build, so the binary is smaller. But it needs to be documented that we use `init`s for that particular reason - so the main func doesn't need to know about handlers packages.

Reply:
> I guess, the idea is that we can exclude various handlers from the build, so the binary is smaller.

I mean, yeah, that's in the PR description :)